backend/routes/auth.js

```diff
@@ -52,5 +54,45 @@ router.post("/signup", async (req, res) => {
   res.status(201).json(newUser);
 });
 
+// login routes!
+
+const rateLimit = require("express-rate-limit");
+
+const loginLimiter = rateLimit({
+  windowMs: 15 * 60 * 1000, // 15 minutes
+  max: 100, // Limit each IP to 10 login attempts per windowMs
```

Comment:
The comment says "10" 🤔 Maybe skip the number in the comment, in case it changes.